library/prelude.lisp

```diff
@@ -84,6 +115,16 @@
    #:mod
    #:even?
    #:odd?)
+  
+  (:import-from
```

Comment:
Don't add dual numbers to the prelude.

Reply:
I added the import because otherwise I get the error: 

```
help: Did you mean COALTON-LIBRARY/MATH/DUAL:DUAL?
```

when I type `(coalton (+ (Dual 2 3) (Dual 3 4)))` inside the `coalton-user` package. it says that `Dual` is an unknown package.

Reply:
Use the `math` nickname, like `(coalton (+ (math:Dual 2 3) (math:Dual 3 4)))`.

Reply:
Thanks